uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/main.controller.js

```diff
@@ -45,6 +47,67 @@ define([
       }
     }
 
+    /**
+     * Displays a case closed success message given the provided case and client
+     * data.
+     *
+     * @param {String} caseTypeTitle - the case type title.
+     * @param {String} clientDisplayName - the display name of the case client.
+     */
+    function displayCaseClosedSuccessMessage (caseTypeTitle, clientDisplayName) {
+      var messageTemplte = 'All tasks in the {Case type} workflow for {Case client} have been completed. Good work!';
+      var message = messageTemplte.replace('{Case type}', caseTypeTitle)
```

Comment:
I see the neat elegance of the code, but at the same time we have some excessive operations here. I would honestly simply use:

```js
'All tasks in the ' + caseTypeTitle + ' workflow for ' + clientDisplayName + ' have been completed. Good work!';
```

I think it is still ok for readability.